pagopa/pn-frontend
# fixed links to selfacare pg board - pn-4250

## Short description
Fixed links to selfcare pg board for pg and login app

## List of changes proposed in this pull request
- Changed buildspec files (pg and login)
- Managed external redirect for users and groups links in pg app

## How to test
- Run login app -> login as pg -> check that the user is redirect to right page
- Run pg app -> click on users and groups links -> check that the user is redirect to the right pages
